services/brig/test/integration/API/Team.hs

```diff
@@ -16,21 +16,19 @@ import Brig.Types.User.Auth
 import Brig.Types.Intra
```

Comment:
This entire module has no logic changes. I just moved some utilities out of it (because I needed them in other tests too) and used `forPooled`.